moxy/eclipselink.moxy.test/src/org/eclipse/persistence/testing/jaxb/dynamic/DynamicJAXBFromXSDTestCases.java

```diff
@@ -929,6 +932,45 @@ public void testXMLSchemaSchema() throws Exception {
         }
     }
 
+    /**
+     * Test for element type reference across multiple XML schemas with different namespaces.
+     * Validates result after marshalling against XML Schema.
+     * @throws Exception
+     */
+    public void testXmlSchemaCrossSchema() throws Exception {
+        InputStream inputStream = ClassLoader.getSystemResourceAsStream(XMLSCHEMA_IMPORT_CROSS_SCHEMA);
+        DynamicJAXBContext jaxbContext = DynamicJAXBContextFactory.createContextFromXSD(inputStream, new CustomEntityResolver(true), null, null);
+        System.setProperty("com.sun.tools.xjc.api.impl.s2j.SchemaCompilerImpl.noCorrectnessCheck", "true");
```

Comment:
this can eventually affect other tests running it the same JVM - you should reset the value in the finally block